index.html

```diff
@@ -0,0 +1,223 @@
+            <a href="#" class="btn btn-primary mt-2">See Project</a>
+          </div>
+        </div>
+      </div>
+      <div class="col-md-4">
+        <div class="card mt-3 text-center ">
+          <img class="card-img-top" src="images/Img-Plaholder.png" alt="Card image">
+          <div class="card-body">
+            <h4 class="card-title">Multi-Post Stories Gain+Glory</h4>
+            <div class="d-flex justify-content-center ">
+              <div class="card bg-light text-dark p-1 m-1"><p><b>Ruby on rails</b></p></div>
+              <div class="card bg-light text-dark p-1 m-1"><p><b>css</b></p></div>
+              <div class="card bg-light text-dark p-1 m-1"><p><b>Javascript</b></p></div>
+              <div class="card bg-light text-dark p-1 m-1"><p><b>html</b></p></div>
+            </div>
+            <a href="#" class="btn btn-primary mt-2">See Project</a>
+          </div>
+        </div>
+        <div class="card mt-3 text-center ">
+          <img class="card-img-top" src="images/Img-Plaholder.png" alt="Card image">
+          <div class="card-body">
+            <h4 class="card-title">Multi-Post Stories Gain+Glory</h4>
+            <div class="d-flex justify-content-center ">
+              <div class="card bg-light text-dark p-1 m-1"><p><b>Ruby on rails</b></p></div>
+              <div class="card bg-light text-dark p-1 m-1"><p><b>css</b></p></div>
+              <div class="card bg-light text-dark p-1 m-1"><p><b>Javascript</b></p></div>
+              <div class="card bg-light text-dark p-1 m-1"><p><b>html</b></p></div>
+            </div>
+            <a href="#" class="btn btn-primary mt-2">See Project</a>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <!-- about section -->
+  <div class="container-fluid about-section">
+    <div class="container-fluid d-flex flex-column text-center mt-5 work-section">
+      <h1>About me</h1>
+      <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+      <div class="row">
+        
+        <div class="col-md-4">
+          <div class="card">
+            <div class="card-body">
+              <img class="aboutme-logo" src="" alt="">
+              <div class="card-title">Languages</div>
+            </div>
+          </div>
+        </div>
+
+        <div class="col-md-4">
+          <div class="card">
+            <div class="card-body">
+              <img class="aboutme-logo" src="" alt="">
+              <div class="card-title">Frameworks</div>
+            </div>
+          </div>
+        </div>
+
+        <div class="col-md-4">
+          <div class="card">
+            <div class="card-body">
+              <img class="aboutme-logo" src="" alt="">
+              <div class="card-title">Skills</div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>    
+  </div>
```

Comment:
The "about me" section doesn't look like your template. Please try to make it look like your template as much as possible.